src/graphql/resolvers/upc-resolvers.ts

```diff
@@ -0,0 +1,18 @@
+import { IResolvers } from '@graphql-tools/utils';
+import { Food, QueryFoodArgs } from '../generated';
+import fetch from 'cross-fetch';
+
+export const FoodResolvers: IResolvers = {
+  Query: {
+    async food(_: void, args: QueryFoodArgs): Promise<Food> {
+      const url = 'https://api.nal.usda.gov/fdc/v1/foods/search?pageSize=2&query='
+      const api_key = '2OZQjK5uKNNJzmdbeK4U2hcz6bPMde4ZNO7SmW4A'
```

Comment:
nit: Could the URL be moved to a file called `src/utils/constants.ts`?
Also, the api_key should ideally be added in as a secret so it couldn't be stolen. Could work on this later as it's not a prod app but we should use something like https://www.npmjs.com/package/dotenv so we don't store API keys in git